### PR TITLE
Create the layout for dashboard 943785328407697

### PR DESCRIPTION
**Pull request by mariashakleva**

### CODE REVIEW

File: resources/sass/app.scss [hunk 1]
```diff
@@ -89,4 +107,10 @@ body{
     &:hover{
     	backgound: $blue;
     }
-}
\ No newline at end of file
+}
+
+// .custom-button{
```
[the remark this answers is not in the document]
Comment (mariashakleva) [reply resources/sass/app.scss]: Everything that is written is used.

[the remark this answers is not in the document]
Comment (davorminchorov) [reply resources/sass/app.scss]: If that's the case, you can remove the commented code

[the remark this answers is not in the document]
Comment (mariashakleva) [reply resources/sass/app.scss]: It's deleted.

File: resources/views/board.blade.php [hunk 2]
```diff
@@ -0,0 +1,53 @@
+@extends('layouts.app')
```
Comment (davorminchorov) [inline resources/views/board.blade.php]: Wasn't this file supposed to be part of another task?

Comment (mariashakleva) [reply resources/views/board.blade.php]: Yes, sorry, I reverted the commits.

**Pull request merged**